malcolm/modules/system/controllers/ProcessController.py

```diff
@@ -64,6 +64,8 @@ def start_ioc(stats, prefix):
     APvPrefix = str
 with Anno("space-separated list of IOCs to monitor"):
     AIocList = str
+with Anno("Use git to parse the YAML version"):
+    AUseGit = bool
```

Comment:
Is this used?

Reply:
Good catch! I'm surprised the linter didn't catch it...